spec/01-unit/08-router_spec.lua

```diff
@@ -4216,6 +4225,7 @@ for _, flavor in ipairs({ "traditional", "traditional_compatible", "expressions"
               {
                 service = service,
                 route = {
+                  id = "e8fb37f1-102d-461e-9c51-6608a6bb8101",
```

Comment:
Why do we need to specify IDs?

Reply:
we need id to add the route to atc-router, see function `add_matcher()`